src/map.rs

```diff
@@ -56,6 +56,8 @@ pub struct Map {
     /// individual tiles may have different sizes. As such, there is no guarantee that this value
     /// will be the same as the one from the tilesets the map is using.
     pub tile_height: u32,
+    /// The length of the side of a hexagonal tile in pixels (Used by tile layers on hexagonal maps).
+    pub hexside_length: Option<u32>,
```

Comment:
I'm guessing it should really be called `hex_side_length`, since "hexside" is not a word.